datasets/strogatz_glider2/metadata.yaml

```diff
@@ -2,6 +2,10 @@
 dataset: strogatz_glider2
 description: |
     This is one state of a 2-state dynamic model for a glider. 
+
+    $\dot{v} = - 0.05 \cdot  v^2 - sin (\theta)$
```

Comment:
```suggestion
    $\dot{v} = - 0.05 \cdot  v^2 - \sin (\theta)$
```

Reply:
oops, I forgot to commit this before merging. added in 90baba4